### DISCUSSION TITLE
Not able to reproduce huggingfaces spaces behavior for table transformation.

### DISCUSSION THREAD

**Discussion by ericmagalhaes** [deepdoctection/deepdoctection#316] [General]
I have tried to replicate your example in huggingface using the standard pipeline, I did the config_overwrite using the parameters you supplied for the custom pipeline you created but it did not work. Would be possible to share an example?

**Comment by JaMe76** [comment]
Please note, as stated in the header of the Huggingface Space, that the models are not open sourced.
